auth-web/tests/unit/views/HomeView.spec.ts

```diff
@@ -0,0 +1,144 @@
+import { createLocalVue, mount } from '@vue/test-utils'
+import BcscPanel from '@/components/auth/BcscPanel.vue'
+import HomeViewDev from '@/views/auth/HomeViewDev.vue'
+import InfoStepper from '@/components/auth/stepper/InfoStepper.vue'
+import LoginBCSC from '@/components/auth/LoginBCSC.vue'
+import TestimonialQuotes from '@/components/auth/TestimonialQuotes.vue'
+import Vue from 'vue'
+import VueRouter from 'vue-router'
+import Vuetify from 'vuetify'
+import Vuex from 'vuex'
+
+Vue.use(Vuetify)
+Vue.use(VueRouter)
+const router = new VueRouter()
+const vuetify = new Vuetify({})
+
+// Prevent the warning "[Vuetify] Unable to locate target [data-app]"
+document.body.setAttribute('data-app', 'true')
+jest.mock('../../../src/plugins/i18n', () => {})
+
+describe('HomeView.vue', () => {
+  let wrapper: any
+  let userModule: any
+
+  beforeEach(() => {
+    const localVue = createLocalVue()
+    localVue.use(Vuex)
+
+    userModule = {
+      namespaced: true,
+      state: {
+        userProfile: {
+        }
+      },
+      actions: {
+        getUserProfile: jest.fn()
+      }
+    }
+
+    const orgModule = {
+      namespaced: true,
+      state: {
+        currentOrganization: {
+        }
+      },
+      actions: {
+        syncOrganizations: jest.fn(),
+        syncCurrentOrganization: jest.fn()
+      }
+    }
+
+    const store = new Vuex.Store({
+      state: {},
+      strict: false,
+      modules: {
+        user: userModule,
+        org: orgModule
+      }
+    })
+
+    wrapper = mount(HomeViewDev, {
+      store,
+      localVue,
+      router,
+      vuetify,
+      mocks: {
+        $t: (mock) => mock
```

Comment:
If I understand correctly, this means there's a function $t which, given a value, returns that value. Why is this needed?

Reply:
It just defines (mocks i should say) `$t` as a global method, otherwise when it tries to render `$t` it has no reference and will return `"TypeError: _vm.$t is not a function"`.

Reply:
> Great job Cam!

lol thanks! This message popped up about 7 times :) Really appreciate it heheh